.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: python
 python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
 
 env:
-  - PLONE_VERSION=4.3 QA=true
+- PLONE_VERSION=4.3
+- PLONE_VERSION=5.0
```

Comment:
@jaroel IMO, it wont make sense; yes, the build will fail but it will not affect the final result and we will have the results available at any time to fix them.